specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+            "in": "body",
+            "type": "string",
+            "description": "Start Date"
+          },
+          {
+            "name":"endDate",
+            "in": "body",
+            "type": "string",
+            "description": "End Date"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "201": {
+            "description": "",
+            "schema": {
+              "properties": {
+                "validationId": {
+                  "type": "string"
+                }
+              }
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ApiSpecsValidation",
+              "description": "Semantic Result"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/tools/apiValidate/result": {
+      "get": {
+        "operationId": "Tools_ApiValidateResult",
+        "parameters": [
+          {
+            "name":"validationId",
+            "in": "query",
+            "type": "string",
+            "description": "Validation Id",
+            "required": true
+          },
+          {
+            "name":"operationId",
+            "in": "query",
+            "type": "string",
+            "description": "Operation Id."
+          },
+          {
+            "name":"env",
+            "in": "query",
+            "type": "string",
+            "description": "Environment"
+          }          
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "404": {
+            "description": "",            
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ApiSpecsValidation",
+              "description": "Semantic Result"
+            }
+          }
+        }
+      }
+    },
+    "/tools/apiValidate/summary": {
+      "get": {
+        "operationId": "Tools_ApiValidateSummary",
+        "parameters": [
+          {
+            "name": "query",
+            "type": "object",
+            "properties": {
+              "validationId": {
+                "type": "string"
+              }
+            }
+          }
+        ],
+        "responses": {
+          "400": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/ErrorObject",
+              "description": "Missing Parameters"
+            }
+          },
+          "404": {
+            "description": "",            
+          },
+          "200": {
+            "description": "",
```

Comment:
TODO: The original apivalidate should return the summary and this would return the entire result